tests/appsec/test_remoteconfiguration.py

```diff
@@ -24,6 +24,10 @@ def _set_and_get_appsec_tags(tracer):
 
 
 def test_rc_enabled_by_default(tracer):
+    try:
+        del os.environ["DD_REMOTE_CONFIGURATION_ENABLED"]
+    except KeyError:
+        pass
```

Comment:
line 27-30 can be replace by one line (using pop with None suppress the Error)
`os.environ.pop("DD_REMOTE_CONFIGURATION_ENABLED", None)`
not sure if it is better, this is only a suggestion.

Reply:
Nice! thank you!